__tests__/intergration.tests.js

```diff
@@ -54,3 +55,15 @@ describe("non existent endpoint", () => {
       });
   });
 });
+
+describe.only("/api", () => {
```

Comment:
Please remove all `.only`'s from tests before submitting code 
If your tests all still pass after doing this you're good to merge 👍